Client/game_sa/CBuildingsPoolSA.cpp

```diff
@@ -48,6 +48,16 @@ inline bool CBuildingsPoolSA::AddBuildingToPool(CClientBuilding* pClientBuilding
     return true;
 }
 
+CClientEntity* CBuildingsPoolSA::GetClientBuilding(CBuildingSAInterface* pGameInterface) const noexcept
+{
+    std::uint32_t poolIndex = (*m_ppBuildingPoolInterface)->GetObjectIndex(pGameInterface);
+
+    if (poolIndex == -1)
```

Comment:
Although `-1` will implicitly be promoted to unsigned, maybe better to explicitly cast it? Just to avoid misreadings in future.

Reply:
Added